multisig/proposals/zksyncmainnet/vip-002/index.ts

```diff
@@ -0,0 +1,202 @@
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS_BRIDGE_ADMIN_PROXY, "updateSendAndCallEnabled(bool)", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "mint(address,uint256)", XVS_BRIDGE_DEST],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "burn(address,uint256)", XVS_BRIDGE_DEST],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS_BRIDGE_ADMIN_PROXY, "setTrustedRemoteAddress(uint16,bytes)", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS_BRIDGE_ADMIN_PROXY, "transferBridgeOwnership(address)", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "migrateMinterTokens(address,address)", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "setMintCap(address,uint256)", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "updateBlacklist(address,bool)", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "pause()", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "unpause()", zksyncmainnet.GUARDIAN],
+    },
+    { target: XVS_BRIDGE_ADMIN_PROXY, signature: "acceptOwnership()", params: [] },
+    {
+      target: XVS_BRIDGE_ADMIN_PROXY,
+      signature: "setTrustedRemoteAddress(uint16,bytes)",
+      params: [LzChainId.bscmainnet, BNB_MAINNET_TRUSTED_REMOTE],
+    },
+    {
+      target: XVS_BRIDGE_ADMIN_PROXY,
+      signature: "setTrustedRemoteAddress(uint16,bytes)",
+      params: [LzChainId.opbnbmainnet, OPBNB_MAINNET_TRUSTED_REMOTE],
+    },
+    {
+      target: XVS_BRIDGE_ADMIN_PROXY,
+      signature: "setTrustedRemoteAddress(uint16,bytes)",
+      params: [LzChainId.ethereum, ETHEREUM_TRUSTED_REMOTE],
+    },
+    {
+      target: XVS_BRIDGE_ADMIN_PROXY,
+      signature: "setTrustedRemoteAddress(uint16,bytes)",
+      params: [LzChainId.arbitrumone, ARBITRUM_ONE_REMOTE],
+    },
+    {
+      target: XVS_BRIDGE_ADMIN_PROXY,
+      signature: "setWhitelist(address,bool)",
+      params: [zksyncmainnet.VTREASURY, true],
+    },
+    {
+      target: XVS_BRIDGE_ADMIN_PROXY,
+      signature: "setWhitelist(address,bool)",
+      params: [zksyncmainnet.GUARDIAN, true],
+    },
+    {
+      target: XVS,
+      signature: "setMintCap(address,uint256)",
+      params: [XVS_BRIDGE_DEST, XVS_MINT_LIMIT],
+    },
+  ]);
```

Comment:
Don't we have to configure the limits between zkSync and the rest of the networks? I think we have to invoke the following functions on the `XVS_BRIDGE_ADMIN_PROXY`, for each network:

* setMaxDailyLimit(uint16,uint256)
* setMaxSingleTransactionLimit(uint16,uint256)
* setMaxDailyReceiveLimit(uint16,uint256)
* setMaxSingleReceiveTransactionLimit(uint16,uint256)

Reply:
I see these functions were invoked during the deployment. Ignore this comment